sunpy/map/tests/test_mapbase.py

```diff
@@ -717,7 +728,7 @@ def test_rotate_new_matrix(generic_map):
 
 
 def test_rotate_rmatrix_angle(generic_map):
-    with pytest.raises(TypeError):
+    with pytest.raises(ValueError, match="You cannot specify both an angle and a rotation matrix."):
```

Comment:
I think this is what was always intended for this test, previously it would `TypeError` because `angle` wasn't a quantity.